.gitignore

```diff
@@ -3,6 +3,7 @@ target/
 !.mvn/wrapper/maven-wrapper.jar
 !**/src/main/**/target/
 !**/src/test/**/target/
+src/test/
```

Comment:
/src/test should never be excluded, because you'll have to write test cases in the future